unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDView.cs

```diff
@@ -423,6 +403,7 @@ public void CleanUp()
         {
             collectiblesItemSelector.OnItemClicked -= controller.WearableClicked;
             collectiblesItemSelector.OnSellClicked -= controller.SellCollectible;
+            collectiblesItemSelector.OnRetryClicked -= controller.RetryLoadOwnedWereables;
```

Comment:
```suggestion
            collectiblesItemSelector.OnRetryClicked -= controller.RetryLoadOwnedWearables;
```

Reply:
Done in https://github.com/decentraland/explorer/pull/2300/commits/a15f01f125788ef879a02ca6446d093b4dced3fb. Thanks!